### PR TITLE
Issue#76

### PR DESCRIPTION
**Pull request by mariadeck**
Adding explanatory text to my expeditions component. This fixes issue#76

### CODE REVIEW

File: src/app/components/navigation/navigation.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
     <md-list ng-show="$ctrl.currentUser">
         <md-subheader class="md-no-sticky">Admin</md-subheader>
-        <md-list-item ng-click="$ctrl.$state.go('expeditions.list')">
+        <md-list-item ng-click="$ctrl.$state.go('expeditions.list')" ng-show="$ctrl.currentUser.userId === $ctrl.currentProject.user.userId">
```
Comment (rodney757) [inline src/app/components/navigation/navigation.html]: this should be visible for any logged in user, not just the project admin

Comment (rodney757) [reply src/app/components/navigation/navigation.html]: This change doesn't do that. It only makes it visible if you are a project admin.

Comment (rodney757) [reply src/app/components/navigation/navigation.html]: You'll need to make another api call to fetch the project members and see if the currentUser is in that list

**Comment by mariadeck** [comment]
The My Expeditions link from nav bar is showing up on my end... are you signed into an account that has any expeditions?

**Comment by rodney757** [comment]
ah, there was a faulty merge by git w/o warning. We both added `$onChanges` to `navigation.component`, so it wasn't working correctly.

I've merged the latest develop into you're branch.

Once you address the comments, I'll merge. Thanks!

File: src/app/components/navigation/navigation.component.js [hunk 2]
```diff
@@ -1,13 +1,31 @@
...
+    if (
+      this.currentUser &&
+      this.currentProject &&
+      'currentProject' in changesObj
+    ) {
+      this.ExpeditionService.getExpeditionsForUser(
+        this.currentProject.projectId,
+        true,
+      ).then(({ data }) => {
+        this.data = data;
```
Comment (rodney757) [inline src/app/components/navigation/navigation.component.js]: I don't think you need this.data, you can simplify this whole thing to  `this.showMyExpeditions = data.length > 0`

**Pull request merged**